projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -1,1296 +1,3379 @@
+	"quark.jei.hint.corundum_cluster_grow": "与对应的刚玉伴生出现，无论前述刚玉是自然生成还是留置生长的。",
+	"quark.jei.hint.corundum_cluster_redirect": "可以重定向信标光束。查看信标物品获取更多信息。",
+	"quark.jei.hint.myalite_crystal_get": "可以在末地外岛附近的螺旋尖顶的顶部找到。",
+	"quark.jei.hint.myalite_crystal_viaduct": "当末影人或者使用末影珍珠的玩家传送到一系列相连的幻境石水晶附近时，他们最终会传送到这一系列水晶的另一端。有分叉也可以正常使用，但出现环状就不行了。",
+	"quark.jei.hint.myalite_crystal_grow": "放置在正在生长的刚玉下，就会开始生长。",
+	"quark.jei.hint.matrix_enchanting": "附魔机制现在使用新的“矩阵附魔”系统。可以使用青金石和经验来制造魔咒碎片，并将其放入网格。合并两个相同类型的魔咒碎片，可以提升魔咒的等级。此外，现在附魔台内部可以保留你的物品和青金石，好耶！",
+	"quark.jei.hint.matrix_influencing": "蜡烛还可以用来增加特定魔咒的获得几率。但如果它下面/它所依附的书架下面是灵魂沙，反而会降低几率。",
+	"quark.jei.hint.repair_item_removed": "该物品的铁砧修复材料已被移除，无法在铁砧中修复。",
+	"quark.jei.hint.repair_item_changed": "该物品的铁砧修复材料已被修改为%s。",
+	"quark.jei.hint.repair_item_changed_multiple": "该物品的铁砧修复材料已被修改为下列材料之一：%s。",
+	"quark.jei.hint.shiba_find_low_light": "如果你手里拿着火把，柴犬会为你指出需要照亮的暗处。",
+
+	"quark.jei.hint.minecraft.clock": "制作出来前不会发挥作用，使利用配方书的有趣逃课方法失效。",
+	"quark.jei.hint.minecraft.amethyst_block": "可以放在音符盒下方，使其发出紫水晶的标志性声音。",
+	"quark.jei.hint.minecraft.shulker_shell": "潜行时双手拿着潜影壳对箱子使用，不用破坏箱子就能将其转换成潜影盒。",
+	"quark.jei.hint.minecraft.calcite": "山地系生物群系中额外加入了方解石的大型团簇。",
+	"quark.jei.hint.minecraft.chain": "和方块连接上之后，该方块被活塞移动时，锁链和另一端的其他东西也会移动。",
+	"quark.jei.hint.minecraft.emerald_block": "可以用来引诱村民跟随你，就像动物跟随食物一样。",
+	"quark.jei.hint.minecraft.jukebox": "使用发射器将唱片放入唱片机，以实现自动播放音乐。",
+	"quark.jei.hint.minecraft.magma_cream": "史莱姆死于岩浆块时，会分裂成岩浆怪。注意：小型岩浆怪不会掉落岩浆膏。",
+	"quark.jei.hint.minecraft.poisonous_potato": "喂给幼年动物后，有几率使其中毒。动物一旦中毒，就说明再也不可能成年了。",
+	"quark.jei.hint.minecraft.scaffolding": "对着脚手架尝试放置方块时，会用该方块替换脚手架链条上最远的一个。潜行时可以正常放置方块。",
+	"quark.jei.hint.minecraft.sponge": "现在可以直接放在水面上。",
+	"quark.jei.hint.minecraft.spore_blossom": "可以使用骨粉来再生。",
+	"quark.jei.hint.minecraft.vine": "现在可以使用剪刀来剪切藤蔓的末端，使其不能进一步生长。",
+	"quark.jei.hint.minecraft.campfire": "玩家用鞘翅掠过时，会受到助推，进一步向上飞行。",
+	"quark.jei.hint.minecraft.soul_campfire": "玩家用鞘翅掠过时，会受到拉扯，向地面坠去。",
+	"quark.jei.hint.minecraft.tinted_glass": "可以挡在信标光束的光路上，使光束变得微弱。这种效果可以叠加。",
+	"quark.jei.hint.minecraft.lava_bucket": "拿起物品栏中的物品，右击熔岩桶就可以永久摧毁该物品！这种方式无法摧毁潜影盒和免疫火焰伤害的物品（如下界合金）。",
+	"quark.jei.hint.minecraft.armor_stand": "现在有手了，可以拿着一件物品再加一面盾牌。",
+
+	"quark.keybind.change_hotbar": "快捷栏交换",
+	"quark.keybind.lock_rotation": "旋转锁定",
+	"quark.keybind.sort_container": "整理容器物品栏",
+	"quark.keybind.sort_player": "整理玩家物品栏",
+	"quark.keybind.transfer_extract": "取出物品",
+	"quark.keybind.transfer_insert": "存入物品",
+	"quark.keybind.patreon_emote.dance": "跳舞（Patreon）",
+	"quark.keybind.patreon_emote.tpose": "T型姿势（Patreon）",
+	"quark.keybind.patreon_emote.dab": "嘻哈超人手势（Patreon）",
+	"quark.keybind.patreon_emote.jet": "冲天姿势（Patreon）",
+	"quark.keybind.patreon_emote.exorcist": "驱魔人姿势（Patreon）",
+	"quark.keybind.patreon_emote.zombie": "模仿僵尸（Patreon）",
+	"quark.keybind.back": "返回",
+	"quark.keybind.camera_mode": "相机模式",
+	"quark.keybind.narrator_readout": "复述游戏内特定信息（较少）",
+	"quark.keybind.narrator_full_readout": "复述游戏内特定信息（较多）",
```

问题在于 Readout 的翻译，和 Full 表何意

---

Readout 之前有个“大声朗读”的奇妙翻译，虽然确实怪，但换到这里可以试试“读出”，“朗读”这类，原表述“复述”也是很可行的。
Full 的翻译应当没问题，“完全”没什么可能，“较多”已经算准确了。

---

> Readout 之前有个“大声朗读”的奇妙翻译，虽然确实怪，但换到这里可以试试“读出”，“朗读”这类，原表述“复述”也是很可行的。

我觉得这里 `Readout` 是个名词（词典上也确实只有名词），读出来的东西，也就是类似 `读数` 的测量结果之类的玩意，至于原翻译 `朗读` 实在太离谱了，复述人本来就是在复述（朗读），疑似把名词换成动词，还疑似同义反复。
而 `复述` 是前面的 `Narrator` 啊，只不过我不直接说 `复述人/复述功能` （的读出结果），而是说（复述人/复述功能） `复述` （了）Readout（此处暂译作 `游戏内特定信息` ）

---

刚刚又看了眼源码，读的东西包括鼠标物品 告示牌 所看方块，有点像waila，其他并没有什么体现机制特别的部分，还是原版辅助功能那种风格。
不过这个是手动触发的，那可以试试 `复述当前内容：部分` 以及 `复述当前内容：全部` ，全部也可以换成较多
```suggestion
	"quark.keybind.narrator_readout": "复述当前内容：部分",
	"quark.keybind.narrator_full_readout": "复述当前内容：全部",
```

---

`当前内容` ……

---

多合理，与 readout 的“读数”相比，不都是“当前”读出来的东西（

---

复述功能：朗读
复述功能：全部朗读